docker/zookeeper/3/debian10/3.5/Dockerfile

```diff
@@ -0,0 +1,93 @@
+FROM marketplace.gcr.io/google/debian10
+
+ARG ZK_VERSION=3.5.8
+
+ENV ZOO_CONF_DIR=/conf \
+    ZOO_DATA_DIR=/data \
+    ZOO_DATA_LOG_DIR=/datalog \
+    ZOO_LOG_DIR=/logs \
+    ZOO_TICK_TIME=2000 \
+    ZOO_INIT_LIMIT=5 \
+    ZOO_SYNC_LIMIT=2 \
+    ZOO_AUTOPURGE_PURGEINTERVAL=0 \
+    ZOO_AUTOPURGE_SNAPRETAINCOUNT=3 \
+    ZOO_MAX_CLIENT_CNXNS=60 \
+    ZOO_STANDALONE_ENABLED=true \
+    ZOO_ADMINSERVER_ENABLED=true
+
+# Add a user with an explicit UID/GID and create necessary directories.
+RUN set -eux; \
+    groupadd -r zookeeper --gid=1000; \
+    useradd -r -g zookeeper --uid=1000 zookeeper; \
+    mkdir -p "${ZOO_DATA_LOG_DIR}" "${ZOO_DATA_DIR}" "${ZOO_CONF_DIR}" "${ZOO_LOG_DIR}"; \
```

Comment:
IMHO better to leave original style in Dockerfile and entrypoint if it comes to variables ${} etc and add only necessary changes. Will be easier to track any updates in upstream.

Reply:
Thanks for the comment but I think with this standard it looks better